pkg/gui/branches_panel.go

```diff
@@ -59,8 +59,22 @@ func (gui *Gui) handleDeleteBranch(g *gocui.Gui, v *gocui.View) error {
 	if checkedOutBranch.Name == selectedBranch.Name {
 		return gui.createErrorPanel(g, "You cannot delete the checked out branch!")
 	}
-	return gui.createConfirmationPanel(g, v, "Delete Branch", "Are you sure you want delete the branch "+selectedBranch.Name+" ?", func(g *gocui.Gui, v *gocui.View) error {
-		if err := gui.GitCommand.DeleteBranch(selectedBranch.Name); err != nil {
+	return gui.createConfirmationPanel(g, v, "Delete Branch", "Are you sure you want to delete the branch "+selectedBranch.Name+" ?", func(g *gocui.Gui, v *gocui.View) error {
+		if err := gui.GitCommand.DeleteBranch(selectedBranch.Name, false); err != nil {
+			return gui.createErrorPanel(g, err.Error())
+		}
+		return gui.refreshSidePanels(g)
+	}, nil)
+}
+
+func (gui *Gui) handleForceDeleteBranch(g *gocui.Gui, v *gocui.View) error {
```

Comment:
I suppose this could be merged into `handleDeleteBranch` with a boolean third argument, similar to `DeleteBranch` in `pkg/commands/git.go`. Would that be preferable?

Reply:
yep that sounds good to me :)